src/main/java/com/amadeus/Response.java

```diff
@@ -97,6 +99,9 @@ private void parseStatusCode() {
 
   // Tries to parse the data
   private void parseData(HTTPClient client) {
+    if (this.statusCode == 204) {
```

Comment:
Instead of having it here with an empty return could it be in the same file line 64 (in the parse method) with something like: 
```java
 protected void parse(HTTPClient client) {
    parseStatusCode();
    if(this.statusCode != 204) {
         parseData(client);
   }
  }
```